cogs/subscribers.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) idoneam (2016-2019)
+#
+# This file is part of Canary
+#
+# Canary is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Canary is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Canary. If not, see <https://www.gnu.org/licenses/>.
+
+# discord-py requirements
+import discord
+from discord.ext import commands
+from discord import utils
+import asyncio
+
+# URL access and parsing
+from bs4 import BeautifulSoup
+
+# Other utilities
+import os
+import re
+import pickle
+import feedparser
+import requests
+
+CFIA_FEED_URL = "http://inspection.gc.ca/eng/1388422350443/1388422374046.xml"
+METRO_STATUS_API = "https://www.stm.info/en/ajax/etats-du-service"
+
+# Default values by line number for status
+# Integers in list are line colours
+METRO_STATUS = {
+    "1": ["Normal métro service", 36431],    # Green Line
+    "2": ["Normal métro service", 15761699],    # Orange Line
+    "4": ["Normal métro service", 16770048],    # Yellow Line
+    "5": ["Normal métro service", 33738]
+}    # Blue Line
+
+try:
+    os.mkdir('./pickles')
```

Comment:
use something like `os.makedirs('./pickles', exist_ok=True)`

Reply:
wontfix on this because it's kind of beyond scope. any changes i made related to the CFIA feed were just file reorganization and no change in function.

Reply:
actually nvm i'll just do it